universe/Effect.h

```diff
@@ -105,6 +107,7 @@ class FO_COMMON_API Effect::EffectsGroup {
     Condition::ConditionBase* Scope() const               { return m_scope; }
     Condition::ConditionBase* Activation() const          { return m_activation; }
     const std::vector<EffectBase*>& EffectsList() const         { return m_effects; }
+    std::string                     GetDescription() const;
```

Comment:
Some comments to explain how GetDescription and DescriptionString are different are needed. Consider renaming one or both?